accounts.go

```diff
@@ -38,6 +39,15 @@ type Field struct {
 	VerifiedAt time.Time `json:"verified_at"`
 }
 
+// Source is a Mastodon account profile field
+type Source struct {
```

Comment:
Is this Source possible to be renamed to AccountSource?

Reply:
Done.